doc/en/user/source/security/layer.rst

```diff
@@ -164,6 +164,9 @@ The mapping of roles to permissions is as follows:
      - r
      - r
 
+.. note:: Specific workspace rule will take precedence over the more generic rule.When *.*.r = * is set and more specific rule for workspace private is assigned as private.*.r = TRUSTED_ROLE,
+	Then specific rule for workspace would take precedence. Hence private workspace layers would be read by TRUSTED_ROLE users only and other users wont be able to read private workspace layers:
+
```

Comment:
The clarification being provided here makes me more confused.  I think this could be better communicated with an example; or a diagram.